README.md

```diff
@@ -7,7 +7,7 @@ This extension ports popular Atom keyboard shortcuts to Visual Studio Code. Afte
 - Multi cursor editing changed from <kbd>alt</kbd> + click to <kbd>ctrl</kbd> (or <kbd>cmd</kbd>) + click. 
 - Enable format on paste. 
 
-All of these features make VS Code more "Atom like." The changes to your User Settings file are as followed. 
+All of these features make VS Code more "Atom like." The changes to your User Settings file (`settings.json`) are as followed. 
```

Comment:
if we are editing this line, it should be "are as follows" (not "followed").

Reply:
done https://github.com/microsoft/vscode-atom-keybindings/pull/66/commits/edd748cf69f53d1f13cee3d912af412fefc302e6

Reply:
done https://github.com/microsoft/vscode-atom-keybindings/pull/66/commits/6d9acc011e9f0d5e691176927be2622e3ebdc514